src/includes/performance/configure-sample-rate/java.mdx

```diff
@@ -35,3 +35,49 @@ Sentry.init { options ->
   }
 }
 ```
+
+## High-Throughput Transport
```

Comment:
why don't we extract this to an includes file? the same content is within 3 files.

Also, I'm not entirely sure this lib is compatible with Android, so it should not be visible for Android, and right now it is

Reply:
In each of the guides this section is slightly different as there is a different way to setup transport in plain Java, Spring and Spring Boot.

Reply:
Android has a separate file for configuring sample rate, so this section does not appear there. Or you meant something else?

Reply:
I (almost) swear I've seen it on https://sentry-docs-git-fork-maciejwalkowiak-gh-java-3333.sentry.dev/platforms/android/performance/
but you're right, my bad, it's Friday!